json_schema_fuzz/__init__.py

```diff
@@ -12,8 +14,15 @@ def random_integer(schema):
 
 def random_object(schema):
     """Generate random JSON object."""
+
```

Comment:
This function is only called for schemas with `"type": "object"`, and `allOf` can occur in any context, so this needs to be restructured a bit.